src/olympia/devhub/utils.py

```diff
@@ -350,6 +350,5 @@ def create_version_for_upload(addon, upload, channel, parsed_data=None):
         # The add-on's status will be STATUS_NULL when its first version is
         # created because the version has no files when it gets added and it
         # gets flagged as invalid. We need to manually set the status.
```

Comment:
Should this comment be changed in some way to indicate that we're not actually manually setting the status, but rather requesting that it be re-evaluated?